glidertest/plots.py

```diff
@@ -1305,3 +1305,38 @@ def plot_ioosqc(data, suspect_threshold=[25], fail_threshold=[50], title='', ax=
         if force_plot:
             plt.show()
     return fig, ax
+
+def plot_max_depth_per_profile(ds: xr.Dataset, bins= 20, **kw: dict) -> tuple({plt.Figure, plt.Axes}):
+    """
```

Comment:
add ax as a parameter
`ax=None `

And add below the 
```
if ax is None:
            fig, ax = plt.subplots()
            force_plot = True
else: 
            fig = plt.gcf()
            force_plot = False

```

Reply:
You can check the other examples and the end you would put also
```
if force_plot:
            plt.show()
```

Reply:
Changed to:
if ax is None:  
            fig, ax = plt.subplots(1,2)  
            force_plot = True 
I hope this is correct, since I want to plots, like in plot_outlier_duration()